### ISSUE TITLE
Cloning customer with paymentMethod from platform to connected account

### ISSUE THREAD

**Issue by taylorchance** [stripe/stripe-node#1321]
Im trying to follow the guide listed here: https://stripe.com/docs/connect/cloning-customers-across-accounts

When I try to create the token I get the error "The customer must have an active payment source attached."

I can't seem to attach a payment method, despite numerous attempts for suggestions online that I won't go into.

When I add the card to the token like so:
```
const token = await stripe.tokens.create({
          customer
          card: payment_method
      },{
          stripeAccount
      });
```

I keep getting the error "Customer cus_xxxxx does not have card with ID pm_xxx", despite validating in the API as well as the dashboard that they do.

I've figured out the issue is that the customer has a default card on hand which id begins with "pm_xxxxxx", I experimented and manually added a new card to test on the stripe Dashboard, this time the card id began with "card_xxxxxx". Creating the token then worked fine. (Despite the fact that the "pm_xxxxxx" card was the default payment method.

This doesn't solve the problem, as the way I have added cards through the API, they begin with "pm_xxxx" and all the payment API calls are based around that.

Any clues? It shouldn't be much of a difference between the 2 cards.

**Comment by remi-stripe** [comment]
@taylorchance Github issues should be limited to bugs with the library itself. Your question is an integration support question better suited for our support team. You can contact them here: https://support.stripe.com/contact

To unblock you though, you are reading the wrong doc, targeted at legacy integrations using the Card API (card_123). You are using the newer API called the PaymentMethods API (pm_123) and should read https://stripe.com/docs/payments/payment-methods/connect

**Comment by taylorchance** [comment]
@remi-stripe I have reached out to stripe support as well. Countless times. They have been useless. I have followed the second document you have linked as well. I can successfully clone the payment method... But when I try to create an invoice or subscription, the API is asking for the customer, not the payment method. I always get the same error "cus_123" does not exist, because it exists on the platform account. So how do I clone a customer to the connected account? I am still very much blocked.

**Comment by remi-stripe** [comment]
@taylorchance This is not a support forum so I'll reply to that one but then I will ask that you continue to work with support directly or use our Discord server for help: https://stripe.com/go/developer-chat

The doc cover this and explain that cloning PaymentMethod is only one step. There is no "link" with the platform customer in any way, you are getting a brand new PaymentMethod object for the connected account, which works exactly the same as a PaymentMethod in the platform. If you want to use it for a Subscription or an Invoice, you first have to create a Customer on the connected account, exactly as you would in your platform. Every account is independent from each other and has its own data. So you would

1/ Clone the PaymentMethod
```
const paymentMethod = await stripe.paymentMethods.create({
  customer: 'cus_123',
  payment_method: 'pm_123',
}, {
  stripeAccount: 'acct_123',
});
```
2/ Create a Customer on the connected account, while attaching the PaymentMethod to that customer and make it the default
```
const customer = await stripe.customers.create({
  email: 'xxxx@yyyy.zzz',
  payment_method: paymentMethod.id,
  invoice_settings: {
    default_ payment_method: paymentMethod.id,
  },
  // any other relevant parameter(s) for that new customer
}, {
  stripeAccount: 'acct_123',
});
```

**Comment by taylorchance** [comment]
@remi-stripe thank you, that helps to unblock me. It would be easier if you could clone a customer with a payment method tho?
